webui/react/src/components/VersionChecker.tsx

```diff
@@ -0,0 +1,62 @@
+import Button from 'hew/Button';
+import { useTheme } from 'hew/Theme';
+import { notification } from 'hew/Toast';
+import { Loadable } from 'hew/utils/loadable';
+import { useObservable } from 'micro-observables';
+import { useEffect } from 'react';
+
+import Link from 'components/Link';
+import { paths } from 'routes/utils';
+import determinedStore from 'stores/determinedInfo';
+import { refreshPage } from 'utils/browser';
+
+const VersionChecker: React.FC = () => {
+  const loadableInfo = useObservable(determinedStore.loadableInfo);
+
+  const {
+    themeSettings: { className: themeClass },
+  } = useTheme();
+
+  // need useEffect to avoid showing notification on each route change:
+  useEffect(() => {
+    /*
+     * Check to make sure the WebUI version matches the platform version.
+     * Skip this check for development version.
+     */
+    Loadable.quickMatch(loadableInfo, undefined, undefined, (info) => {
+      if (!process.env.IS_DEV && info.version !== process.env.VERSION) {
+        const btn = (
+          <Button type="primary" onClick={refreshPage}>
+            Update Now
+          </Button>
```

Comment:
do we still need this button? `Update Now` button doesn't update anything, just refreshing the page. feels like its misleading.

---

not from this PR, but `refreshPage` keep appending the query param. i dont think its expected

Reply:
fixed `refreshPage` issue. 

It's a good point about the button. Actually, I think the release notes link doesn't make sense either because it'll point to the current version's docs, which won't include "what's new".

I'm hesitant to remove anything in this bugfix PR though. Maybe need separate ticket for Product to define how to improve this notification?

Reply:
> I'm hesitant to remove anything in this bugfix PR though. Maybe need separate ticket for Product to define how to improve this notification?

Im worried not to have discussion after this pr is merged, so it might be good to add the change here, but up to you.

Reply:
created https://hpe-aiatscale.atlassian.net/browse/CM-581